app/src/main.js

```diff
@@ -135,6 +138,16 @@ app.on('ready', () => {
   }
 });
 
+app.on('ready', () => {
+  installExtension(REACT_DEVELOPER_TOOLS)
+    .then((name) => {
+      console.log("Successfully installed", name)
+    })
+    .catch((err) =>{
+      console.log('An error occurred when trying to add extention: ', err)
```

Comment:
It seems strange to be doing this inside of the render process (is that correct?). A limitation of this is that we wouldn't be checking at packaging time if a chrome extension is valid or not. I think `electron-devtools-installer` might force this though 😕